### PR TITLE
[stable/sentry] Update version sentry to 9.1.1

### LINKED ISSUE #12947: [stable/sentry] update version to 9.1.0
**Is this a BUG REPORT or FEATURE REQUEST?** (choose one): FEATURE REQUEST

> If this is a FEATURE REQUEST, please:
 > - Describe *in detail* the feature/behavior/change you'd like to see.

Sentry [released version 9.1.0](https://github.com/getsentry/sentry/releases/tag/9.1.0), also available from [Docker Hub](https://hub.docker.com/_/sentry). It would be great if it was supported by the Chart.

**Which chart**: stable/sentry

**What happened**: Version 9.0.0 [is used](https://github.com/helm/charts/blob/12aad70/stable/sentry/values.yaml#L6)

**What you expected to happen**: Version 9.1.0 is used

### PR DESCRIPTION
**Pull request by batazor**
Signed-off-by: Login Victor <batazor111@gmail.com>

#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #12947

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/sentry/Chart.yaml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: Sentry is a cross-platform crash reporting and aggregation platform.
 name: sentry
-version: 1.3.6
-appVersion: 9.0
+version: 1.3.9
```
Comment (okgolove) [inline stable/sentry/Chart.yaml]: 1.4.0

**Review by okgolove — CHANGES_REQUESTED** [review]
Hello. Thank you for the PR.
Rebase your branch please.

File: stable/sentry/Chart.yaml (hunk 1 above)
Comment (batazor) [reply stable/sentry/Chart.yaml]: Hi! Done

**Comment by batazor** [comment]
@okgolove I updated the version in README.md

**Pull request merged**